src/halmos/__main__.py

```diff
@@ -1318,21 +1330,17 @@ def parse_build_out(args: HalmosConfig) -> Dict:
             continue
 
         for json_filename in os.listdir(sol_path):  # for each contract name
-            try:
-                if not json_filename.endswith(".json"):
-                    continue
-                if json_filename.startswith("."):
-                    continue
+            if not json_filename.endswith(".json"):
+                continue
+            if json_filename.startswith("."):
+                continue
 
-                json_path = os.path.join(sol_path, json_filename)
-                with open(json_path, encoding="utf8") as f:
-                    json_out = json.load(f)
+            json_path = os.path.join(sol_path, json_filename)
+            with open(json_path, encoding="utf8") as f:
+                json_out = json.load(f)
```

Comment:
pulling this out of the try..except block to reduce the scope of the try blocks and minimize nesting